.github/workflows/pr-1-ci.yml

```diff
@@ -0,0 +1,153 @@
+name: PR Checks
+on:
+  pull_request:
+    branches:
+      - 'release-*'
+      - 'dev-*'
+    paths-ignore:
+      # These are ignored because they don't have anything to do with the model itself
+      - .github/**
+      - tools/**
+      - doc/**
+      - .*
+      - metadata.yaml
+      - README.md
+jobs:
+  commit-check:
+    name: Commit Check
+    # We run this job to check if the current commit was done during a workflow run.
+    # Such as when 'github-actions' bumps the metadata.yaml file or updates the checksums
+    # in the `testing` directory.
+    # This is so we don't recursively commit and check infinitely during this workflow.
+    runs-on: ubuntu-latest
+    outputs:
+      authorship: ${{ steps.head-commit.outputs.authorship }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: HEAD Commit Authorship
+        id: head-commit
+        run: echo "authorship=$(git log -1 --pretty=format:'%an')" >> $GITHUB_OUTPUT
+
+  repro-ci:
+    # run the given config on the deployment GitHub Environment (`environment-name`) and
+    # upload the checksums and test details
+    needs:
+      - commit-check
+    if: needs.commit-check.outputs.authorship != 'github-actions'
+    uses: access-nri/reproducibility/.github/workflows/checks.yml@main
```

Comment:
This is calling an external action which utilises an environment variable `EXPERIMENT_LOCATION`

https://github.com/ACCESS-NRI/reproducibility/blob/main/.github/workflows/checks.yml#L55

But the environment sets `EXPERIMENTS_LOCATION` (note the plural)

https://github.com/ACCESS-NRI/access-om2-configs/settings/environments/2030732480/edit

to be `/scratch/tm70/repro-ci/experiments`.

Is this an error? Should it be `EXPERIMENT_LOCATION`?

Reply:
I don't think it's an error. 
`vars.EXPERIMENTS_LOCATION` (defined in https://github.com/ACCESS-NRI/access-om2-configs/settings/environments/2030732480/edit) is used for all experiments regardless of model or config tag. 
`env.EXPERIMENT_LOCATION` (defined here in `ACCESS/NRI/reproducibility`: https://github.com/ACCESS-NRI/reproducibility/blob/cda63e38eba91222b48f2704aa32364a1fbd5ad0/.github/workflows/checks.yml#L41) is a more specific path that builds upon the `vars.EXPERIMENTS_LOCATION`, adding the model name and config tag subdirectories.